### PR TITLE
[fix] Embed Quantinuum logo in notebooks

### PR DESCRIPTION
**Pull request by ferbetanzo**
The Quantinuum logo is now correctly embedded at the top of the example notebooks. This is done by using the `data:image/svg+xml` in a markdown cell (thx @yao-cqc for the pointer!), and providing the URL encodede `<svg>`. Instead of adding this to each notebook, this embedded logo lives in `logo_header_markdown.md`, and is appended at the top of each notebook in the `check-examples` script.

If you modify the notebook generation scripts (in `examples/python`), make sure to re-generate the corresponding .ipynb by appending the logo at the top, but do not check in the modified python with the header string (otherwise the logo would appear 2 times).

### CODE REVIEW

**Comment by ferbetanzo** [comment]
It seems github does not show the embedded svg correctly when previewing the .ipynb file, has this been always the case?
image

File: examples/check-examples [hunk 1]
```diff
@@ -8,10 +8,25 @@ do
...
     rm ${name}-gen.ipynb
-    # Run script:
-    ./answer_input_prompts ${file} $PYTKET_REMOTE_QUANTINUUM_USERNAME $PYTKET_REMOTE_QUANTINUUM_PASSWORD
+    rm python/${file}
+
+    # Restore original
+    mv python/${name}-original.py python/${file}
+
+    # Run script
+    #./answer_input_prompts ${file} $PYTKET_REMOTE_QUANTINUUM_USERNAME $PYTKET_REMOTE_QUANTINUUM_PASSWORD
```
Comment (cqc-alec) [inline examples/check-examples:31]: Was this meant to be commented out?

**Review by cqc-alec — COMMENTED** [review]
I think we could do with a `README.md` in the `examples` directory, explaining that the notebooks are genetated from the python files, with any gotchas for contributors.

**Comment by cqc-alec** [comment]
> It seems github does not show the embedded svg correctly when previewing the .ipynb file, has this been always the case?

I'm not sure.

File: examples/check-examples (hunk 1 above)
Comment (ferbetanzo) [reply examples/check-examples:31]: Good catch! I'll add the README.md :D

**Comment by yao-cqc** [comment]
> It seems github does not show the embedded svg correctly when previewing the .ipynb file, has this been always the case?
> image

It doesn't render in VS code either.

image

**Comment by yao-cqc** [comment]
https://github.com/CQCL/pytket-quantinuum/blob/display-logo-in-notebook/examples/Quantinuum_circuit_submissions.ipynb

I've managed to display the logo by first converting the svg logo to png, and then using the base 64 encode of that png in the markdown cell. Using a base64 encode of the svg doesn't seem to work and I don't know why.

**Comment by cqc-alec** [comment]
> https://github.com/CQCL/pytket-quantinuum/blob/display-logo-in-notebook/examples/Quantinuum_circuit_submissions.ipynb
> 
> I've managed to display the logo by first converting the svg logo to png, and then using the base 64 encode of that png in the markdown cell. Using a base64 encode of the svg doesn't seem to work and I don't know why.

Can we use the PNG instead of SVG?

**Comment by ferbetanzo** [comment]
The png does not look as bad as I thought:
image

**Comment by ferbetanzo** [comment]
@cqc-alec or @yao-cqc, do you think this is ok now?

**Comment by cqc-alec** [comment]
Any idea why the "Check example notebooks" test is failing?

**Review by cqc-alec — APPROVED** [review]

**Review by yao-cqc — APPROVED** [review]
Looks good. The logo is a little low res, we should ask for a better logo and make an update in a future PR.

**Pull request merged**